compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -221,58 +225,15 @@ object Inliner {
 
   /** Replace `Inlined` node by a block that contains its bindings and expansion */
   def dropInlined(inlined: Inlined)(using Context): Tree =
+    val topLevelClass = Some(inlined.call.symbol.topLevelClass).filter(_.exists)
```

Comment:
Maybe try
```suggestion
    val topLevelClass = 
      if inlined.call.isEmpty then None
      else Some(inlined.call.symbol.topLevelClass)
```
In theory, the symbol would not exist only if the `inlined.call` is the EmptyTree.